ZionEscape/Node.h

```diff
@@ -24,7 +24,9 @@ ref class Node {
     this->hCost = 0;
     this->parent = nullptr;
   }
-
+  ~Node() {
+    delete parent;
```

Comment:
Antes de hacer `delete`, primero evalúa si parent no es ya `nullptr`.

```suggestion
    if (parent != nullptr)
      delete parent;
```